Source/Core/InputCommon/GCAdapter.cpp

```diff
@@ -462,8 +776,8 @@ GCPadStatus Input(int chan)
 
 	{
 		std::lock_guard<std::mutex> lk(s_mutex);
-		std::copy(std::begin(s_controller_payload), std::end(s_controller_payload),
-			std::begin(controller_payload_copy));
+		const u8 *controller_payload = Fetch(tp);
+		std::copy(controller_payload, controller_payload + 37, controller_payload_copy);
```

Comment:
why are we copying using a magic number instead of `std::begin` and `std::end` as was done previously?